Simplenote/CollaborateViewController.swift

```diff
@@ -38,7 +38,7 @@ class ShareViewController: NSViewController {
 
 // MARK: - Private
 //
-private extension ShareViewController {
+private extension CollaborateViewController {
 
     func startListeningToNotifications() {
```

Comment:
So, if I understand this corretly ... what we're doing here is --  
_Stop_ (sharing, instead) _Collaborate_ and _Listen_ to notifications?

Reply:
I've renamed `Share > Collaborate` because the names were confusing, to begin with! 

Sharing involves using the Share Sheet (which is all about sending content elsewhere: another app, the web).
On the other hand, Collaboration is all about multiple users editing the same document.

(For the record!) The Collaboration UI is a popover that shows up in the tags area, and displays the legend `type an email here to share this note`.

Thanks for stepping by sir!!

Reply:
@jleandroperez ... it was a Vanilla Ice reference 😆

Reply:
Ahhhhh dammit!!! LOOOL